chall06/pcuadrad.c

```diff
@@ -0,0 +1,260 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+static int		isinset(char c, char charset)
```

Comment:
I think this function would be better formatted as a `#DEFINE` condition, but I think it's pretty subjective.